### DISCUSSION TITLE
Can I import from Obsidian.md?

### DISCUSSION THREAD

**Discussion by mgook** [athensresearch/athens#1339] [Question / Answer]
Hi. I want to migrate from obsidian to athens.
Is it possible??

**Comment by yifanyin** [comment]
It is not possible now. From my experience, the writing style one develops in Obsidian is really different from block-based Athens. If you really want to migrate, I'd suggest you manually move each note and restructure the content along the way.

**Reply by joelhans** [reply]
:+1: 

And here's the discussion about import/export for anyone wanting to weigh in or give the whole feature request an upvote: https://github.com/athensresearch/athens/discussions/874
